include/swift/AST/FileUnit.h

```diff
@@ -101,6 +101,11 @@ class FileUnit : public DeclContext {
                  ObjCSelector selector,
                  SmallVectorImpl<AbstractFunctionDecl *> &results) const = 0;
 
+  /// Find all SPI imported from \p importedModule by this module, collecting
+  /// their identifiers in \p spis.
+  virtual void lookupImportedSPIs(const ModuleDecl *importedModule,
+                                  SmallVectorImpl<Identifier> &spis) const {};
```

Comment:
The function name is kind of confusing. Can we change it to something like `lookupImportedSPIKinds` or `lookupImportedSPINotions`.

Reply:
I changed the ambiguous naming for an "SPI name" to "SPI group" instead.